www/index.html

```diff
@@ -59,6 +59,12 @@
     <script src="app/menu/route.js"></script>
     <script src="app/menu/controller.js"></script>
 
+    <!-- Music Search -->
+    <script src="app/musicsearch/module.js"></script>
+    <script src="app/musicsearch/route.js"></script>
+    <script src="app/musicsearch/controller.js"></script>
+    <script src="app/musicsearch/service.js"></script>
+
     <!-- Home -->
     <script src="app/home/module.js"></script>
     <script src="app/home/route.js"></script>
```

Comment:
@pfernandez01 Everything looks pretty good, just a couple of styling fixes :smile: Let me know when those are ready so I can merge this.